cmd/app/add_test.go

```diff
@@ -356,11 +414,84 @@ func TestAppAddCommand(t *testing.T) {
 				cm.API.AssertCalled(t, "DeveloperAppInstall", mock.Anything, mock.Anything, mockOrgAuth.Token, mock.Anything, mock.Anything, mock.Anything, "T123", mock.Anything)
 			},
 		},
+		// TODO(semver:major): Remove this test when the defaulting to deployed is removed.
+		"adds a new deployed app when team flag is provided and environment flag is not set": {
```

Comment:
note: Test for defaulting to `deployed` when the team flag is specified and there is no `--environment` flag.